third_party/build/bin/grpc_build.sh

```diff
@@ -64,13 +65,16 @@ git submodule update --init
 # change default prefix from /usr/local to /tmp/build-grpc-dev/install/usr/local
 sed -i 's/.usr.local$/\/tmp\/build-grpc-dev\/install\/usr\/local/' Makefile
 
-# build and install grpc
+mkdir -p cmake/build
+pushd cmake/build
+cmake ../.. -DgRPC_INSTALL=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=${WORK_DIR}/install/usr/local -DgRPC_ABSL_PROVIDER=module -DgRPC_CARES_PROVIDER=package -DgRPC_PROTOBUF_PROVIDER=module -DgRPC_RE2_PROVIDER=module -DgRPC_SSL_PROVIDER=OpenSSL
```

Comment:
The default prefix is already changed in line 66. So is `-DCMAKE_INSTALL_PREFIX=${WORK_DIR}/install/usr/local` a duplication to that?